src/overview/assets/index.html

```diff
@@ -196,6 +199,9 @@ <h3 class="font-weight-light">Spring</h3>
         <div class="row mb-3">
           <div class="col">
             <h3 class="font-weight-light">Microservices</h3>
+            <div>
+              <a href="command:java.helper.openUrl?%22https%3A%2F%2Fmarketplace.visualstudio.com%2Fitems%3FitemName%3Dredhat.vscode-quarkus%26ssr%3Dfalse%23overview%22" title="Marketplace link for Quarkus Tools for VS Code">Quarkus Tools for VS Code</a>
```

Comment:
Please put this recommendation behind Docker because Quakus is still new and we are not sure about the demand/popularity yet.

And usually, the link points to the documentation page which helps users quickly started and further explore. I'm not sure whether the marketplace page serves that purpose but I'll leave it to you guys to judge.

Reply:
And would you like to have an "Install Quakus Extension" link just like what Docker has?

Reply:
Since we don't have an up-to-date wiki for the extension yet, we've decided to link the marketplace page for now, since it still provides information on the features. I'll be sure to make a new PR when we do have a wiki/guide for the extension.

Thanks for the suggestion, I've added an "Install Quarkus Extension" link to keep things consistent with the Kubernetes and Docker links.